deebot_client/events/water_info.py

```diff
@@ -25,3 +32,4 @@ class WaterInfoEvent(Event):
     amount: WaterAmount
     # None means no data available
     mop_attached: bool | None = field(kw_only=True, default=None)
+    sweep_type: SweepType
```

Comment:
```suggestion
    sweep_type: SweepType | None = None
```

To mark it as optional